google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -376,13 +385,21 @@ def _store_iam_policy(self, policy):
 
             # binding['members'] can have duplicate ids
             members = set(binding['members'])
+            db_members = []
             for member in members:
-                member = member.replace(':', '/', 1)
+                member = member.replace(':', '/', 1).lower()
 
                 # We still might hit external users or groups
                 # that we haven't seen in gsuite.
-                if member not in self.member_cache and \
-                        member not in self.member_cache_policies:
+                if member in self.member_cache and member not in db_members:
```

Comment:
the if elif here is a little bit confusing. Can you clear the logic here a little bit?

Reply:
I agreed it's confusing to have this elif statement. I have updated to have the second if statement to check for both                 `member not in self.member_cache and member not in self.member_cache_policies` instead